pkg/cache/sql/store/store.go

```diff
@@ -180,6 +181,11 @@ func (s *Store) Add(obj any) error {
 	}
 
 	err = s.upsert(key, obj)
+	if err != nil {
+		log.Errorf("Error in Store.Add for type %v: %v", s.name, err)
+		return err
+	}
+	return nil
 	return err
```

Comment:
The suggestion snippet ended up not getting the return err so now we have two returns here, sorry!

```suggestion
	return nil
```

Reply:
🤦 did not check, sorry. fixed now